### PR TITLE
INBOX-1443 

### PR DESCRIPTION
**Pull request by ThiagoYU**
Fixed record creation bug described in [INBOX-1443](https://ns1inc.atlassian.net/browse/INBOX-1443)

### CODE REVIEW

File: rest/model/data/meta.go [hunk 1]
```diff
@@ -302,7 +302,14 @@ func MetaFromMap(m map[string]interface{}) *Meta {
...
 			case "Subdivisions":
-				fv.Set(reflect.ValueOf(v.(map[string]interface{})))
+				switch v.(type) {
+				case string:
+					var subMap map[string]interface{}
+					json.Unmarshal([]byte(v.(string)), &subMap)
+					fv.Set(reflect.ValueOf(subMap))
+				case map[string]interface{}:
+					fv.Set(reflect.ValueOf(v.(map[string]interface{})))
+				}
```
Comment (Zach-Johnson) [inline rest/model/data/meta.go:312]: can we add a test for this case?

**Review by Zach-Johnson — CHANGES_REQUESTED** [review]
Needs a test

**Review by ponbiki — DISMISSED** [review]
LGTM (just see Zach's comment regarding tests)

**Review by Zach-Johnson — APPROVED** [review]

**Comment by Zach-Johnson** [comment]
@ThiagoYU I'm getting test failures when I run the tests for this locally, I think they are unrelated though. We should fix them in a separate PR I think?

```rest/model/pulsar/pulsar_job_test.go:9:6: TestNewBBPulsarJob redeclared in this block
	/Users/zachjohnson/w/ns1/opensource/ns1-go/rest/model/pulsar/application_test.go:8:28: previous declaration
ok  	gopkg.in/ns1/ns1-go.v2/mockns1	0.451s
--- FAIL: TestApplication (0.00s)
    --- FAIL: TestApplication/Get (0.00s)
        --- FAIL: TestApplication/Get/Error (0.00s)
            --- FAIL: TestApplication/Get/Error/HTTP (0.00s)
                application_test.go:107:
                    	Error Trace:	application_test.go:107
                    	Error:      	"application does not exist" does not contain "test error"
                    	Test:       	TestApplication/Get/Error/HTTP
--- FAIL: TestPulsarJob (0.00s)
    --- FAIL: TestPulsarJob/List (0.00s)
        --- FAIL: TestPulsarJob/List/Error (0.00s)
            --- FAIL: TestPulsarJob/List/Error/Other_errors (0.00s)
                pulsar_job_test.go:95:
                    	Error Trace:	pulsar_job_test.go:95
                    	Error:      	"pulsar application does not exist" does not contain "test error"
                    	Test:       	TestPulsarJob/List/Error/Other_errors
FAIL
FAIL	gopkg.in/ns1/ns1-go.v2/rest	0.357s
```

**Comment by ThiagoYU** [comment]
> 

Yes @Zach-Johnson 
This is unrelated, we can take a look at this in other PR.

**Pull request merged**